src/stem.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
loving the modularity (new file)

since we have two files, may as well get OCD and move them into a sub-package/sub-directory